### ISSUE TITLE
used variables but got an error

### ISSUE THREAD

**Issue by liuseen-l** [sweepline/eslint-plugin-unused-imports#67]
<img width="512" alt="image" src="https://github.com/sweepline/eslint-plugin-unused-imports/assets/91084928/0eb68aee-9a91-4d1d-bb93-5db675885d80">
<img width="923" alt="image" src="https://github.com/sweepline/eslint-plugin-unused-imports/assets/91084928/08f07c5e-e0b7-418e-b62e-b3d0b3b2db95">

**Comment by sweepline** [comment]
The first steps to helping you is posting your config and making sure it is not reported when not using this plugin. So you disable this package and reenable `eslint/no-unused-vars` or `@typescript-eslint/no-unused-vars`. If it is still reported it is a problem upstream. This package does not really defined `no-unused-vars` and is just using the parent function.

**Comment by liuseen-l** [comment]
```js
import antfu from '@antfu/eslint-config'

export default antfu(
  {
    formatters: true,
    typescript: true,
    vue: true,
    jsonc: true,
    yaml: true,
    jsx: true,
    markdown: true,
    ignores: [
      ' **/dist',
      '**/node_modules',
      '**/*.svg',
      '**/cache',
    ],
  },
  [
    {
      files: ['playgrounds/**'],
      rules: {
        'no-console': 'off',
      },
    },
    {
      files: ['packages/core/src/v-debounce/index.ts'],
      rules: {
        'eslint/no-unused-vars': 'off',
        '@typescript-eslint/no-unused-vars': 'off',
      },
    },
  ],
)

```

`npx eslint .` still report error
` 20:5  error  'el' is assigned a value but never used. Allowed unused vars must match /^_/u  unused-imports/no-unused-vars`

**Comment by sweepline** [comment]
I do not really know how this antfu config works. It seems like they already include it in their package, so maybe there is a way to configure it within their setup? I recon you should try to ask in [their repo](https://github.com/antfu/eslint-config/) instead.
